lib/tools/profiler.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+const v8Profiler = require('v8-profiler');
+const EventEmitter = require('events');
+
+// 30 min
+const DEFAULT_INTERVAL_MS = 1800000;
+const NAME_PREFIX = 'v8.heap.snapshot';
```

Comment:
I prefer to expose ways to override in the constructor, these are just internal defaults. Overriding these defaults is an implementation dependent detail. I think we can easily also add them to the `.env.example` but use them where we instantiate the profiler, rather than in the profiler itself.  I'm not currently exposing NAME_PREFIX, so that is a change I should make. Does that makes sense?